lib/browser/api/menu-item-roles.js

```diff
@@ -0,0 +1,146 @@
+const roles = {
+  about: {
+    get label () {
+      const {app} = require('electron')
+      return `About ${app.getName()}`
+    }
+  },
+  close: {
+    label: 'Close',
+    accelerator: 'CmdOrCtrl+W',
+    windowMethod: 'close'
+  },
+  copy: {
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    webContentsMethod: 'copy'
+  },
+  cut: {
+    label: 'Cut',
+    accelerator: 'CmdOrCtrl+X',
+    webContentsMethod: 'cut'
+  },
+  delete: {
+    label: 'Delete',
+    accelerator: 'Delete',
+    webContentsMethod: 'delete'
+  },
+  front: {
+    label: 'Bring All to Front'
+  },
+  help: {
+    label: 'Help'
+  },
+  hide: {
+    get label () {
+      const {app} = require('electron')
```

Comment:
Yeah, this requires reworking `app.js` a bit since this at the top would currently fail because of a circular require between app, menu, and menu item.